### ISSUE TITLE
Is Sony's LevelEditor useful?

### ISSUE THREAD

**Issue by gwald** [horde3d/Horde3D#238]
I was playing around with this level editor... and thought it might be useful for horde3D.
https://github.com/SonyWWS/LevelEditor?tab=readme-ov-file

My thoughts:

- It's more [mature and advanced](https://github.com/SonyWWS/LevelEditor/wiki/LevelEditor-Features-and-Benefits).
- Supports collada (not the full subset)
- XML scene based.
- It's easy to build and maintain (C#/dotnet)
- It was used in AAA games.

**Comment by algts** [comment]
I've looked at it before, but there are several points to consider:
- Requires a large amount of time to adapt the editor to use Horde3D
- Unmaintained - you'll have to maintain it yourself (large unknown code base, probably with many quirks)
- Probably uses an unmaintained .NET version - would have to be ported to current .net core
- Windows only (don't know about the window toolkit they are using - if it is Windows Forms, it may somewhat work under Linux, if WPF - good luck)
- Horde's C# bindings may not be optimal for the editor, would also require some work

To sum it up, it is probably a lot of work :) I won't say it cannot be done, but with current Horde's development pace it is probably an impossible task.

**Comment by gwald** [comment]
Good points,  I dont see it as a h3d editor replacement, but a nice complement.

I tried it in linux, I couldn't get it running, and there's no mention in the repo of linux. :/
https://github.com/SonyWWS/LevelEditor/issues/42

I was thinking of starting small... comparing the H3D scene xml to it's Sony's .lvl XML to see if it translates.
Then Hardcode a H3D scene file exporter, I'm new to C# so I'll see how far I get :D
